### PR TITLE
Add a Workflow that automatically add a comment based on labels

### PR DESCRIPTION
**Pull request by seokho-son**
### Describe your changes
This PR add a Workflow that automatically add a comment based on labels.

### Related issue number or link (ex: `resolves #issue-number`)
- Requested feature from the latest CNCF Glossary Working Group meeting.
https://docs.google.com/document/d/1kjNrLWbMHSQ3x4ob8Q_lNOoJ4f8uPQcuD9a0Sif24NY/edit?pli=1

> Add an automation for every incoming PR that says something like: “To see where this is in the review pipeline and follow the progress, please have a look at the [project board](https://github.com/orgs/cncf/projects/26)”

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by seokho-son** [comment]
FYI. Tested in https://github.com/m-cmp/mc-infra-manager/pull/7

image

**Comment by seokho-son** [comment]
PTAL: @jihoon-seo @iamNoah1

File: .github/auto-comment-config.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+label_mappings:
+  - labels: 
+      - hold
+    comment: Wait, please do not proceed this yet.
```
Comment (nate-double-u) [inline .github/auto-comment-config.yaml:4]:
```suggestion
    comment: Wait, please do not proceed with this yet.
```

File: .github/auto-comment-config.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - labels: 
+      - lang/en
+    comment: To see where this is in the review pipeline and follow the progress, please have a look at the [project board](https://github.com/orgs/cncf/projects/26/views/1).
```
Comment (nate-double-u) [inline .github/auto-comment-config.yaml:7]:
```suggestion
    comment: To see where this is in the review pipeline and follow the progress, please look at the [definition review board](https://github.com/orgs/cncf/projects/26/views/1).
```

**Review by nate-double-u — COMMENTED** [review]
Just a couple tweeks, thanks for this @seokho-son

File: .github/auto-comment-config.yaml (hunk 2 above)
Comment (seokho-son) [reply .github/auto-comment-config.yaml]: Thanks @nate-double-u for the enhancement!
Applied your comment as suggested!
- https://github.com/cncf/glossary/pull/2389/files#diff-0d826e18f815cafca3a28b7926fc6430b2ba6d52ad4289656a5201e2561dafe1R4
- https://github.com/cncf/glossary/pull/2389/files#diff-0d826e18f815cafca3a28b7926fc6430b2ba6d52ad4289656a5201e2561dafe1R7

**Review by iamNoah1 — APPROVED** [review]
nice, lgtm

**Review by nate-double-u — APPROVED** [review]
thanks @seokho-son, /lgtm

**Pull request merged**